reference-implementation/to-upstream-wpts/writable-streams/bad-strategies.https.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/service-workers/service-worker/resources/test-helpers.sub.js"></script>
+<script src="/streams/resources/test-initializer.js"></script>
```

Comment:
Sure. What do you think of a follow-up commit to change all of them to use the absolute URL though? It's not actually a file that's local in this repo like `recording-streams.js` is.